site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -36,6 +36,26 @@ The same could be run for `alpha.swagger.json` and `beta.swagger.json` as requir
 
 You can read more about OpenAPI/Swagger code generation in their [Command Line Tool Documentation](https://swagger.io/docs/open-source-tools/swagger-codegen/)
 
+## SDK Functionality
+
+| Area            | Action              | Implemented |
+|-----------------|---------------------|-------------|
+| Lifecycle       | Ready               | ✔️          |
+| Lifecycle       | Health              | ✔️          |
+| Lifecycle       | Reserve             | ✔️          |
+| Lifecycle       | Allocate            | ✔️          |
+| Lifecycle       | Shutdown            | ✔️          |
+| Configuration   | WatchGameServer     | ✔️          |
```

Comment:
It also has GetGameServer https://github.com/googleforgames/agones/blob/main/sdks/swagger/sdk.swagger.json#L52

Reply:
Included it